ci/dockerfile.ci

```diff
@@ -17,7 +17,7 @@ RUN cd /core && git pull origin main && pip install . --no-deps
 RUN cd /nvtabular && git pull origin main && pip install . --no-deps
 RUN cd /models && git pull origin main && pip install . --no-deps
 RUN cd /systems && git pull origin main && pip install . --no-deps
-ENV PYTHONPATH=/usr/lib/python3.8/site-packages:/usr/local/hugectr/lib
+ENV PYTHONPATH=/usr/local/hugectr/lib
```

Comment:
Should this maintain any PYTHONPATH that existed beforehand?

Reply:
None other than hugectr... everything is installed into dist-packages and site-packages.